2015-10-27.md

```diff
@@ -0,0 +1,32 @@
+Hi Tesselators!
+
+It's been a few weeks since the last This Week in Tessel, our apologies! We've been hard at work on the Tessel repos, building and growing the project.
```

Comment:
Definitely. We should have an update checklist though.. so that in tandem with this newsletter, the website is updated as well. Does such a thing exist?